pages/docs/reference/functions.md

```diff
@@ -43,13 +43,16 @@ Stream().read() // create instance of class Stream and call read()
 
 ### Parameters
 
-Function parameters are defined using Pascal notation, i.e. *name*: *type*. Parameters are separated using commas. 
+Function parameters are defined using Pascal notation, i.e. *name*: *type*. Parameters are separated using commas. Use a [trailing comma](coding-conventions.html#trailing-commas) when declaring function parameters.
```

Comment:
Not sure we can just recommend it in general for all functions. There are hundreds of examples without it in our own docs and Kotlin codebase :)